compiler/src/main/java/com/dylibso/chicory/compiler/internal/Compiler.java

```diff
@@ -561,7 +562,7 @@ private static void emitFunction(
                         null);
 
         // uncomment if you ever want to troubleshoot invalid bytecode generation
-        // methodWriter = new CheckMethodAdapter(methodWriter);
+        methodWriter = new CheckMethodAdapter(methodWriter);
 
```

Comment:
I assume you uncommented this to aid development.

Reply:
Yes, that's a day saver, thanks for it!
Good catch!